api/api/src/test/scala/uk/ac/wellcome/platform/api/services/WorksServiceTest.scala

```diff
@@ -269,6 +269,49 @@ class WorksServiceTest
       )
     }
 
+    it("Should simple_query_syntax into account and not fail when it's invalid") {
+      val workEmu = createIdentifiedWorkWith(
+        title = "a b c"
+      )
+
+      // This would land up with a too_many_clauses error due to the precedence syntax
```

Comment:
This sentence feels incomplete. “This would land up” – but it doesn’t, so we must be doing something to stop it!

Also, “the precedence syntax” isn’t very clear – unless you know how ES syntax works, it’s not obvious that `()` are precedence operators. Would be useful to have something that explains what originally happened, and what we’ve changed to ensure it doesn’t crash here.